CreatubblesAPIClient/Sources/Requests/Gallery/MyGalleriesRequest.swift

```diff
@@ -25,7 +25,7 @@
 
 import UIKit
 
-enum MyGalleriesRequestFilter {
+public enum GalleriesRequestFilter {
     case none
```

Comment:
Does it make sense to actually have a `none` filter? If a request isn't filtered, then shouldn't the filter be simply nil? (I see that that's exactly how it works)
Maybe .unknown would make more sense?

Reply:
hmm I think I added it due to objc functionality